opendatahub-io/notebooks
# Switch base image to UBI for AMD rocm install

## Description

Switch base image to UBI for AMD rocm install
Related-to: https://issues.redhat.com/browse/RHOAIENG-7501

## How Has This Been Tested?




```
1. build the base `podman build -t amd-base .`
2. check the necessary bits are available in amd-base , with `rpm -qa`
```

## Merge criteria:
